src/handlers/review_submitted.rs

```diff
@@ -0,0 +1,46 @@
+use crate::github::{Issue, IssueCommentAction, IssueCommentEvent, Label, Comment, PullRequestReviewState};
+use crate::{config::ReviewSubmittedConfig, github::Event, handlers::Context};
+
+pub(crate) async fn handle(
+    ctx: &Context,
+    event: &Event,
+    config: &ReviewSubmittedConfig,
+) -> anyhow::Result<()> {
+    if let Event::IssueComment(
+        event
+        @
+        IssueCommentEvent {
+            action: IssueCommentAction::Created,
+            issue: Issue {
+                pull_request: Some(_),
+                ..
+            },
+            ..
+        },
+    ) = event
+    {
+        if event.comment.pr_review_state == PullRequestReviewState::Approved {
+            return Ok(());
+        }
+
+        if event.issue.assignees.contains(&event.comment.user) {
```

Comment:
I think we should only do this for "Request Changes". I'm not sure how to interpret a simple "comment" review that has no "inclination".

Reply:
Makes sense, changed it